README.md

```diff
@@ -481,7 +481,7 @@ Values have the following effects:
 * 'absent': Ensures that the VM doesn't exist on Azure..
 
 #####`name`
-*Required* The name of the virtual machine.
+*Required* The name of the virtual machine. The name may have at most 64 characters. Some images may have more restrictive requirements.
```

Comment:
See the fix from @aharden  and the hint in https://github.com/puppetlabs/puppetlabs-azure/blob/master/spec/support/context/with_known_names.rb#L3 .